geometry_msgs/Vector3 Ta
std_msgs/Float64 phi_a
std_msgs/Float64 theta_a
std_msgs/Float64 psi_a
geometry_msgs/Vector3 Tb
std_msgs/Float64 phi_b
std_msgs/Float64 theta_b
std_msgs/Float64 psi_b
